.vscode/extensions.json

```diff
@@ -1,8 +0,0 @@
-{
-    "recommendations": [
-        "DavidAnson.vscode-markdownlint",
-        "ms-python.python",
-        "ms-dotnettools.csharp",
-        "ms-vscode.powershell",
-    ]
-}
```

Comment:
I dunno why these files are deleted.

Reply:
This was an accidental mistake. I've restored them now :)